src/main/java/com/mini/test/Service/MapService.java

```diff
@@ -0,0 +1,11 @@
+package com.mini.test.Service;
```

Comment:
패키지 네이밍 컨벤션 맞춰야되고, 패키지 이름에 test는 빼는게 좋아.
```suggestion
package com.mini.zomboid.service;
```

Reply:
넵